pkg/remote/aws/api_gatewayv2_api_enumerator.go

```diff
@@ -0,0 +1,46 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type ApiGatewayV2ApiEnumerator struct {
+	repository repository.ApiGatewayV2Repository
+	factory    resource.ResourceFactory
+}
+
+func NewApiGatewayV2ApiEnumerator(repo repository.ApiGatewayV2Repository, factory resource.ResourceFactory) *ApiGatewayV2ApiEnumerator {
+	return &ApiGatewayV2ApiEnumerator{
+		repository: repo,
+		factory:    factory,
+	}
+}
+
+func (e *ApiGatewayV2ApiEnumerator) SupportedType() resource.ResourceType {
+	return aws.AwsApiGatewayV2ApiResourceType
+}
+
+func (e *ApiGatewayV2ApiEnumerator) Enumerate() ([]*resource.Resource, error) {
+	apis, err := e.repository.ListAllApis()
+	if err != nil {
+		return nil, remoteerror.NewResourceListingError(err, string(e.SupportedType()))
+	}
+
+	results := make([]*resource.Resource, 0)
```

Comment:
Can you add a capacity here so that we know the maximum length of the slice?

`results := make([]*resource.Resource, 0, len(apis))`